src/Infrastructure/Symfony/Command/GeocodeCommand.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Symfony\Command;
+
+use App\Application\GeocoderInterface;
+use App\Application\GeographyFormatterInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class GeocodeCommand extends Command
+{
+    public function __construct(
+        private GeocoderInterface $geocoder,
+        private GeographyFormatterInterface $geographyFormatter,
+    ) {
+        parent::__construct('app:geocode');
+    }
+
+    public function configure(): void
+    {
+        $this->addArgument('address', InputArgument::REQUIRED, 'The address as a string');
```

Comment:
Rajouter un bloc descriptif  via `$this->help('...');` ? Cf ma remarque du dessus, et puis, lorsque tu affiches la liste des commandes via `bin/console`, ça t'affiche également la description de la commande.

Reply:
Je viens de voir que tu pouvais utiliser les attributs PHP pour déclarer une commande : 
```
#[AsCommand(
    name: 'app:my-command',
    description: 'Description.',
    hidden: false,
)]
```

Reply:
Ouip, j'avais hésité à utiliser cette annotation puisqu'ailleurs on n'en utilise pas, mais c'est un truc spécifique Symfony ici donc logique

Reply:
On utilise les annotations sur le routing également pour définir une route